vendor/modules.txt

```diff
@@ -97,9 +97,12 @@ github.com/tidwall/pretty
 github.com/xdg/scram
 # github.com/xdg/stringprep v0.0.0-20180714160509-73f8eece6fdc
 github.com/xdg/stringprep
-# golang.org/x/crypto v0.0.0-20190530122614-20be4c3c3ed5
+# github.com/youmark/pkcs8 v0.0.0-20201027041543-1326539a0a0a
```

Comment:
I wasn't sure on our process for adding dependencies... I ran `go get github.com/youmark/pkcs8`, and `go mod vendor` after using the new package.